Jorge/HTML & CSS/index.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>First Task</title>
+</head>
+<body>
+  <div class="main-container">
+    <nav class="header">
+      <div class="header__logo">
+        <img src="images/broly.png" alt="logo">
+      </div>
+      <div class="header__menu">
+        <ul>
+          <li><a href="#">FEATURES</a></li>
+          <li><a href="#">TEMPLATES</a></li>
+          <li><a href="#">INTEGRATION</a></li>
+          <li><a href="#">PRICING</a></li>
+        </ul>
+      </div>
+      <div class="header__button">
+        <button class="button">SIGN IN</button>
+        <button class="button primary">SIGN UP</button>
+      </div>
+    </nav>
+    <div class="hero">
+      <h3>Hola, me llamo</h3>
+      <h1>Jorge</h1>
+    </div>
+    <section class="achivements">
+      <div class="achivements__item">
+        <span>10</span>
+        Años de experiencia
+      </div>
+      <div class="achivements__item">
+        <span>10</span>
+        Galardones
+      </div>
+      <div class="achivements__item">
+        <span>10</span>
+        Clientes satisfechos
+      </div>
+      <div class="achivements__item">
+        <span>10</span>
+        Proyectos concluidos
+      </div>
+    </section>
+    <section class="technologies">
+      <article class="technologies__blog">
+        <h2>THE AMAZING <br>HUMBLE</h2>
+        <h2>Spend less Time Coding</h2>
+        <article class="blog-content">
+          <div class="blog-content__img">
+            <img src="images/goku-coding.png" alt="blog-image">
+          </div>
+          <div class="blog-content__description">
+            <section>
+              <h3>Become a power-user</h3>
+              <p>
+                If you spend a good percentage of your Time
+                coding in a specific enviroment, you should
+                know exactly how to control it.
+              </p>
+            </section>
+            <section>
+              <h3>Know how to use Vim</h3>
+              <p>
+                If you spend a good percentage of your Time
+                coding in a specific enviroment, you should
+                know exactly how to control it.
+              </p>
+            </section>
+          </div>
+        </article>
+      </article>
+      <article class="technologies__stack">
+        <h4>Stack:</h4>
+        <div class="stack-item">
+          <img src="images/esfera.png" alt="stack">
```

Comment:
This image doesn't transmit any message as it is being used as an icon or a bullet in your list, so you have 2 choices, you either use a div and set the image as background or if you want to keep using the img tag then leave then set the alt to the following: `alt=""`, this way the screenreader will ignore reading out loud this image